server/routes/utils/dataReportHelpers.ts

```diff
@@ -6,7 +6,7 @@
 import esb, { Sort } from 'elastic-builder';
 import converter from 'json-2-csv';
 import _ from 'lodash';
-import moment from 'moment';
+import moment from 'moment-timezone';
```

Comment:
Does this need to be added to `package.json`?

Reply:
it's in core